frontend/src/containers/Routing/index.tsx

```diff
@@ -28,6 +28,7 @@ import RespondentReport from "../ReportPage/RespondentReport";
 import Profile from "../../components/Profile";
 import RedirectFormEditor from "../../components/RedirectFormEditor";
 import ErrorPage from "../ErrorPage";
+import NewsList from "../../components/NewsList";
 
 // just for demo
```

Comment:
Do we have something here that we need just for demo, or it is just an old comment?